bm-instance-agent/bm_instance_agent/systems/linux/distro/kylin.py

```
@@ -16,10 +21,16 @@ class KylinDriver(CentOSDriver):
 
     def attach_port(self, instance_obj, network_obj):
         for port in network_obj.ports:
-            if port.type == port.PORT_TYPE_BOND:
-                raise exception.NewtorkInterfaceConfigParasInvalid(
-                    exception_msg="port type {} is not support".format(port.type))
             self._attach_port(port)
+            if port.type == port.PORT_TYPE_BOND \
+                    and port.mac == instance_obj.provision_mac:
+                self._write_bond_to_dracut_config(port)
+                try:
+                    subprocess.check_call(['dracut', '-f'])
```

Ruff (0.14.10)

Starting a process with a partial executable path

(S607)